lib/linguist/languages.yml

```diff
@@ -303,6 +305,7 @@ Bison:
   extensions:
   - .bison
   ace_mode: text
+  color: "#6A463F"
```

Comment:
OK. Now I think Bison does not need a color, too. But maybe in the future it may be. In my opinion, let the color stay.